packages/strapi-parts/src/Dialog/Dialog.stories.mdx

```diff
@@ -0,0 +1,49 @@
+<!--- Dialog.stories.mdx --->
+
+import { useState } from 'react';
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import { Text } from '../Text';
+import { Stack } from '../Stack';
+import { Row } from '../Row';
+import { Button } from '../Button';
+import { Dialog } from './Dialog';
+import { DialogBody } from './DialogBody';
+import { DialogFooter } from './DialogFooter';
+import AlertWarningIcon from '@strapi/icons/AlertWarningIcon';
+import DeleteIcon from '@strapi/icons/DeleteIcon';
+
+<Meta title="Dialog" component={Dialog} />
+
+# Dialog
+
+This is the doc of the `Dialog` component
+
+## Dialog
+
+Description...
+
+<Canvas>
+  <Story name="base">
+    <Dialog onClose={() => {}} title="Confirmation" labelledBy="confirmation">
```

Comment:
It would be better if we could trigger the modal from a button in the stories. Right now the modal prevents you from accessing the story